src/services/EthereumService.ts

```diff
@@ -136,6 +138,8 @@ export class EthereumService {
       throw new Error(`Please connect to either ${Networks.Mainnet} or ${Networks.Rinkeby}`);
     }
 
+    if (isCypress) return;
+
     // comment out to run DISCONNECTED
```

Comment:
@dkent600 

I wanted to run the tests disconnected, to reduce noise from outside.
Let me know what you think, as this can be easily reverted.

Reply:
> the ability to run connected?

Clarifying my intention.
I wanted to reduce the amount of connections, until a test is implemented for a particular connection.
Ie. it is unclear to me (lack of knowledge about Deals and web3), how the "connections" affect the app.
So I wanted to write tests until I encounter a problem, where I _need_ to enable the connections again.

Anyway, I'll remove this, as your points are also valid.